proxy/http/Http1ClientSession.cc

```diff
@@ -427,9 +427,12 @@ Http1ClientSession::release(ProxyClientTransaction *trans)
     ka_vio = this->do_io_read(this, INT64_MAX, read_buffer);
     ink_assert(slave_ka_vio != ka_vio);
 
-    // Y!?
-    // client_vc->add_to_keep_alive_lru();
-    client_vc->cancel_active_timeout();
+    if (client_vc) {
+      client_vc->add_to_keep_alive_queue();
+      if (client_vc) {
```

Comment:
Why not just cancel the active timeout first and then add to the keep alive queue?